wfe/web-front-end.go

```diff
@@ -1147,6 +1246,64 @@ func (wfe *WebFrontEndImpl) Certificate(response http.ResponseWriter, request *h
 	return
 }
 
+// CertificateRequest is used by clients to request the latest certificate issued under
+// a cached CSR.
```

Comment:
It's not really a cached CSR, right? They're going to a URL (which happens to match the serial) and they get a cert.